includes/batch.inc

```diff
@@ -212,10 +212,11 @@ function islandora_paged_content_create_pdf_paged_content_batch_operation($paged
  *   The batch definition.
  */
 function islandora_paged_content_create_ocr_batch(AbstractObject $paged_content, array $pages, array $options) {
-  return array(
+  $return = array(
     'operations' => array(
       array('islandora_paged_content_create_ocr_batch_operation', array(
-          $pages, $options)),
+        $pages, $options)),
+      array('islandora_paged_content_create_ocr_paged_content_batch_operation', array($paged_content->id)),
```

Comment:
Should this be here? Seems to be accounted for in the `$options` check below... While this would _always_ add it...